pycqed/measurement/pulse_sequences/multi_qubit_tek_seq_elts.py

```diff
@@ -980,10 +981,8 @@ def n_qubit_tomo_seq(
         pulse_list.extend(ro_pulses)
 
         if preselection:
-            ro_pulses_presel = generate_mux_ro_pulse_list(qubit_names, 
-                                                          operation_dict,
-                                                          'RO_presel',
-                                                          True, -ro_spacing)
+            ro_pulses_presel = generate_mux_ro_pulse_list(
+                qubit_names, operation_dict, 'RO_presel', 'end', -ro_spacing)
```

Comment:
I am suprised that the preselection code was buggy for so long (I checked that the fourth argument to the function should not have been a boolean even in August), good to have it fixed.
Maybe `True` still worked somehow.

But wouldn't it be more intuitive if `ro_spacing` was the timing between the starts  of the two readouts?
For that the third argument would have to be changed to `'start'`.

Reply:
Should I also change the default value of this input parameter in the generate_mux_ro_pulse_list function? This might break some measurements like the multi_parity_multi_round_seq or others that are not even in the master.

Reply:
I think the default should stay as it is, because in the typical case where we want to read out the qubits at the end of the experiment, the `ref_point='end'` makes most sense.
Only for preselection we want to set `ref_point='start'` and `pulse_delay=-ro_spacing`

Reply:
I see. Ok

Reply:
I found the commit from June that changed the `first_pulse_start=False` argument to `ref_point='end'` argument.
So `first_pulse_start=True` should translate to `ref_point='start'`

Reply:
One gets an error though when one tries to run the measurement. Because the function expects a strong instead of a bool?

Reply:
could be because of that, yes.
I am not sure what `Segment` does when the ref-point is set to a bool

Reply:
Probably fails:

![image](https://user-images.githubusercontent.com/14844492/77530112-251a1100-6e91-11ea-8682-405884cbc652.png)